src/components/MoneyRequestReportView/MoneyRequestReportActionsList.tsx

```diff
@@ -28,6 +28,7 @@ import ONYXKEYS from '@src/ONYXKEYS';
 import type * as OnyxTypes from '@src/types/onyx';
 import type Transaction from '@src/types/onyx/Transaction';
 import MoneyRequestReportTransactionList from './MoneyRequestReportTransactionList';
+import SearchMoneyRequestReportEmptyState from './SearchMoneyRequestReportEmptyState';
```

Comment:
Why is this name prefixed with `Search`? Won't we show this empty state everywhere once the UI is ready?

Reply:
this is an empty state already with completed data for this particular case, so that we do not have to enter the same parameters in two places